modules/reference/pages/rpk/rpk-group/rpk-group-delete.adoc

```diff
@@ -1,23 +1,13 @@
 = rpk group delete
 
-Delete groups from brokers.
+Delete consumer groups
 
-Older versions of the Kafka protocol included a retention_millis field in
-offset commit requests. Group commits persisted for this retention and then
-eventually expired. Once all commits for a group expired, the group would be
-considered deleted.
+Redpanda supports deleting consumer groups manually through `rpk group delete` to manage offsets proactively. This is useful for cleaning up offsets sooner than the automatic cleanup, such as when creating temporary groups for investigations or testing.
 
-The retention field was removed because it proved problematic for infrequently
-committing consumers: the offsets could be expired for a group that was still
-active. If clients use new enough versions of OffsetCommit (versions that have
-removed the retention field), brokers expire offsets only when the group is
-empty for offset.retention.minutes. Redpanda does not currently support that
-configuration (see #2904), meaning offsets for empty groups expire only when
-they are explicitly deleted.
+Group offset deletion can happen through:
 
-You may want to delete groups to clean up offsets sooner than when they
-automatically are cleaned up, such as when you create temporary groups for
-quick investigation or testing. This command helps you do that.
+* Periodic Offset Expiration: Offsets expire automatically when the group has been empty for a set duration. See xref:reference:properties/cluster-properties.adoc#group_offset_retention_sec[Cluster properties - `group_offset_retention_sec`]
+* Kafka `OffsetDelete` API: Offsets can be explicitly deleted using the Kafka `OffsetDelete` API.
```

Comment:
I wasn't able to find the official Kafka or Confluent docs for this. Is this the equivalent: https://docs.redpanda.com/current/reference/rpk/rpk-group/rpk-group-offset-delete/ and should we just point readers to that? cc @twmb or @r-vasquez ?

Reply:
It is equivalent, probably better to redirect

Reply:
Added a link to the rpk command at the end of the sentence.